lua/ulx/modules/sh/cfc_propify.lua

```diff
@@ -58,7 +58,7 @@ function cmd.unpropifyPlayer( ply )
     local prop = ply.ragdoll
     ply.ragdoll = nil
 
-    if not IsValid( prop ) then
```

Comment:
why this change? this seems like it would be worse than the previous line
IsValid( prop ) wont error if prop is nil
prop:IsValid( prop) will error if prop is  nil

Reply:
I'd prefer IsValid( prop ) but for some unholy reason, unragdoll fails to work correctly unless I do this, even though this file doesn't interfere with ragdoll/unragdoll